package.json

```diff
@@ -23,10 +23,15 @@
     "rollup": "rollup -c",
     "test": "jest --coverage && eslint src",
     "test:watch": "jest --coverage --watchAll --json --outputFile=testOutput.json",
+    "test-storybook": "test-storybook",
+    "test-storybook:watch": "test-storybook --watch",
+    "test-storybook:browsers": "test-storybook --browsers firefox chromium webkit",
     "publish": "npm run rollup"
```

Comment:
Run this command:
`npm i copyfiles @svgr/cli  --save-dev`

AND 

Add these commands to the scripts object:
```
    "clean": "rm -rf ./dist",
    "copy-fonts": "copyfiles -u 1 src/config/fonts/* dist/",
    "compile-icons": "npx @svgr/cli -d src/icons src/icons --typescript",
    "prebuild": "npm run clean && npm run compile-icons",
    "postbuild": "npm run copy-fonts",
```

Reply:
I am working on these scripts.